src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void toDrawFigure() {
+        System.out.println("Figure: circle, area: " + getArea() + " sq. units, "
+                + "radius: " + radius + " units, color: " + color);
```

Comment:
The method name 'toDrawFigure' does not follow the convention of not using verbs for method names. Consider renaming it to something more appropriate like 'draw'. Additionally, the method should be part of a separate interface as per the checklist point about not putting all behavior into a single interface if the methods are conceptually different from each other.